apps/native-component-list/src/screens/AccelerometerScreen.tsx

```diff
@@ -34,15 +41,15 @@ export default class AccelerometerScreen extends React.Component {
 
   componentDidMount() {
     Accelerometer.addListener(({ x, y }) => {
-      this.state.items.forEach((box, index) => {
+      this.state.items.forEach((_, index) => {
         // All that matters is that the values are the same on iOS, Android, Web, ect...
 
         const { perspective } = this.props;
         const nIndex = index + 1;
 
         Animated.spring(this.state.items[index].position, {
           toValue: {
-            x: (x.toFixed(1) * perspective * nIndex) / COUNT,
+            x: (+x.toFixed(1) * perspective * nIndex) / COUNT,
```

Comment:
🤔

Reply:
[`toFixed` return value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed#Return_value).
It was just tricky explicit conversion between `String` and `Number` 😉